ui/components/src/EChart/EChart.tsx

```diff
@@ -14,7 +14,8 @@
 import React, { useEffect, useLayoutEffect, useRef } from 'react';
 import { ECharts, EChartsCoreOption, init, connect } from 'echarts/core';
 import { Box, SxProps, Theme } from '@mui/material';
-import { isEqual, debounce } from 'lodash-es';
+import isEqual from 'lodash/isEqual';
+import debounce from 'lodash/debounce';
```

Comment:
I _think_ you don't need the nested imports in all use cases (e.g. I think webpack may be smart enough these days to tree shake with normal imports), but using them to be extra safe.

Reply:
Yeah I'm still don't understand what the "correct" way to use lodash is, this approach is what I landed on [in core too](https://github.com/perses/perses/blob/v0.33.0/ui/core/src/utils/memo.ts#L15) when I needed the fix for e2e's in #1165